Jenkinsfile

```diff
@@ -167,15 +167,16 @@ stage('Configure') {
 					new JdkBuildEnvironment(version: '17', testCompilerTool: 'OpenJDK 17 Latest',
 							condition: TestCondition.BEFORE_MERGE,
 							isDefault: true),
-					new JdkBuildEnvironment(version: '18', testCompilerTool: 'OpenJDK 18 Latest',
-							condition: TestCondition.AFTER_MERGE),
 					// We want to enable preview features when testing early-access builds of OpenJDK:
 					// even if we don't use these features, just enabling them can cause side effects
 					// and it's useful to test that.
 					new JdkBuildEnvironment(version: '19', testCompilerTool: 'OpenJDK 19 Latest',
 							testLauncherArgs: '--enable-preview',
```

Comment:
Just wondering if we should remove the preview flag from 19 based on the comment above... (19 not being EA build anymore)

Reply:
The preview flag is about preview features; those can still be preview in GA releases. The goal is for the JDK team to be able to release features without committing to backwards compatibility.

So, in short, the flag is still useful :)

Reply:
I should probably change the comment though.

Reply:
Done, thanks for the heads-up